lib/components/user/styled.ts

```diff
@@ -48,6 +48,22 @@ export const TripPanelHeading = styled(Panel.Heading)`
   background-color: white !important;
 `
 
+export const TripPanelAlert = styled(Button)`
+  border: none;
+  color: red;
+  cursor: pointer;
+  float: right;
+  text-decoration: underline;
+
+  :hover,
+  :focus,
+  :active {
+    background-color: transparent;
+    color: #d9534f;
```

Comment:
Once we switch over to using an `a` (and therefore don't need to override bootstrap's style sheets), I'd recommend making this an `opacity: 90%` on hover instead of a different color!